rir/src/compiler/debugging.h

```diff
@@ -0,0 +1,44 @@
+#ifndef PIR_DEBUGGING
+#define PIR_DEBUGGING
+
+#include "utils/EnumSet.h"
+
+namespace rir {
+namespace pir {
+
+#define LIST_OF_PIR_PRINT_DEBUGGING_FLAGS(V)                                   \
```

Comment:
Add some documentation here? My main concern is that someone will update the list here but then forget to update `rir.R`.